readthedocs/index.rst

```diff
@@ -21,7 +21,7 @@ Introduction
 
 Matchms allows to easily build custom spectra processing pipelines and to compute spectra similarities.
 
-.. image:: _static/flowchart_matchms.png
+.. image:: ../_static/flowchart_matchms.png
```

Comment:
Spectrum plot is OK, but now flowchart is kaput
```suggestion
.. image:: _static/flowchart_matchms.png
```

Reply:
Yeah... guess I fixed it kaput.